README.md

```diff
@@ -1,6 +1,83 @@
-# visitAfrica-backend
-
-# Badges 
 
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed%20by-Hound-blueviolet)](https://houndci.com)
 [![Build Status](https://travis-ci.org/Soma-Technologies-Inc/visitAfrica-backend.svg?branch=develop)](https://travis-ci.org/Soma-Technologies-Inc/visitAfrica-backend)
+[![Coverage Status](https://coveralls.io/repos/github/andela/blackninjas-backend/badge.svg?branch=develop)](https://coveralls.io/github/Soma-Technologies-Inc/visitAfrica-backend?branch=develop)
```

Comment:
Put badges under the description of the project!

Reply:
Yeah _ am gonna look on that_